livekit-ffi/protocol/room.proto

```diff
@@ -100,6 +100,32 @@ message PublishDataCallback {
   optional string error = 2;
 }
 
+// Publish transcription messages to room
+message TranscriptionSegment {
+  string id = 1;
+  string text = 2;
+  uint64 start_time = 3;
+  uint64 end_time = 4;
+  bool final = 5;
+}
+
+message PublishTranscriptionRequest {
+  uint64 local_participant_handle = 1;
+  string participant_identity = 2;
+  string track_id = 3;
+  repeated TranscriptionSegment segments = 4;
+  string language = 5;
+}
```

Comment:
Nit but I keep all the requests in the same "code region"

Reply:
you mean without the newlines?

Reply:
no sorry I mean, the code I selected should be moved at the end of this file

Reply:
Hrmm really? The PublishDataRequest (and friends) are right above this

Reply:
the selection is wrong. I mean for Transcription and TranscriptionSegment, the requests are in the right position